lib/spack/spack/package.py

```diff
@@ -2373,6 +2377,8 @@ def use_cray_compiler_names():
     os.environ['F77'] = 'ftn'
 
 
+# Note this is a feature required for external packages maintained by at least
```

Comment:
I think it might be useful to also explain the supported use case (if you happen to know).

Reply:
FYI. I've asked Greg (@becker33) if there's anything I can add to the comment.

Reply:
Some codes require that all of their third party libraries are installed in a single location. Calling this in the install method can create that location.

Reply:
I've iterated on the comment based on feedback.